packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -643,7 +647,8 @@ export class RunningSummarizer implements IDisposable {
 						cancellationToken,
 					);
 					const result = await resultSummarize.receivedSummaryAckOrNack;
-
+					const summarizeResult = await resultSummarize.summarySubmitted;
+					this.eventEmitter?.emit("experimentalSummary", summarizeResult);
```

Comment:
This is where the magic is. If we want to segregate errors from successes, we can change the name and emit summarizeResult in success for happy paths and summarizeResult after for unhappy paths.